analyzer/src/main/kotlin/managers/GradleDependencyHandler.kt

```diff
@@ -28,6 +28,7 @@ import org.eclipse.aether.repository.RemoteRepository
 
 import org.ossreviewtoolkit.analyzer.managers.utils.DependencyHandler
 import org.ossreviewtoolkit.analyzer.managers.utils.MavenSupport
+import org.ossreviewtoolkit.model.Identifier
```

Comment:
Commit message:

> [1] ./cli/src/main/kotlin/commands/AnalyzerCommand.kt:166

Would be nice if this was a permalink, i.e. https://github.com/oss-review-toolkit/ort/blob/c596bc2b0f77e138ed9e6ee38dc55108e7c48fbb/cli/src/main/kotlin/commands/AnalyzerCommand.kt#L166